script/Arcueid/Day 5/05_00_ARC05_2.txt

```diff
@@ -96,7 +96,7 @@ Ancestors are something above that... They can churn out those feudal lords with
 [sha:2a887582af4a54e5bb05f5f20460fb40cf63bdf3]{
 -- Page 181, Offset 9026.
 -- 「死徒たちの世界は二十七の勢力に分けられているの。二十七の国があると思えばいいわ。
-"The world of Dead Apostles is divided into twenty-seven powers. You can think of it as twenty-seven different countries.
+"The Dead Apostle world is divided into twenty-seven major powers. You can think of it as twenty-seven different countries.
```

Comment:
I tend to like reformulating "world of..." because... of my own biases?

It probably doesn't make too much of a difference, but for example looking up "the world of sports" vs "the sports world" on google you get almost an exact equal amount of results, SLIGHTLY favoring "the sports world"

My main inspiration is how they formulate it in Harry Potter "the wizarding world", not "the world of wizardry"

Somehow that feels like the better fantasy approach.

Reply:
I get the opposite sense to be honest. When I think "the world of" it calls more to mind like "the world of magic" or "the world of mystery". It has a more... whimsical(?) tone

Reply:
Somehow when I look at it "the Dead Apostle world" just feels so much more right. I don't know, it bugs the shit out of me when its "the world of Dead Apostles".

But at the end of the day, it appears this difference is basically hair-splitting so if you to feel that "the world of" is better then let's just leave it at that.